packages/next/server/on-demand-entry-handler.ts

```diff
@@ -414,6 +414,11 @@ export function normalizePage(page: string) {
   if (unixPagePath === '/index' || unixPagePath === '/') {
     return '/'
   }
+
+  if (!unixPagePath.startsWith('/')) {
+    return '/' + unixPagePath.replace(/\/index$/, '')
```

Comment:
Seems that this is not the right solution, all page paths should start with a `/` so we should throw when someone tries to call render with a value like `'test'` instead of `'/test'`

Reply:
Ok, will update my PR with that fix instead. Thanks for the feedback!